@types/chalk/index.d.ts

```diff
@@ -0,0 +1,426 @@
+/**
```

Comment:
This works but do we like it? 😄

Reply:
I personally don't want to spend any more time on this shit...the maintainers don't seem to want to change it, so until there is a more sustainable solution I'm fine with this.

At least we don't block updates to other deps.

Reply:
Yeah it's super frustrating.